base64ct/src/decoder.rs

```diff
@@ -26,7 +26,7 @@ const CHAR_LF: u8 = 0x0a;
 /// Stateful Base64 decoder with support for buffered, incremental decoding.
 ///
 /// The `E` type parameter can be any type which impls [`Encoding`] such as
-/// [`Base64`] or [`Base64Unpadded`].
+/// [Base64][`crate::variant::standard::Base64`] or [Base64Unpadded][`crate::variant::standard::Base64Unpadded`].
```

Comment:
Check line 17 of this file. There are imports of these, although they should probably be gated on `#[cfg(doc)]` instead of `#[cfg(docsrs)]`.